### DISCUSSION TITLE
Недоступные треки

### DISCUSSION THREAD

**Discussion by jooblex** [Chimildic/YaMuTools#99] [Вопрос/ответ]
Я правильно понимаю недоступные треки - это те, которые пропали после санкций и т.д. Как их можно посмотреть?

**Comment by Chimildic** [comment]
В коллекции, вкладка треки, фильтр все. Недоступные треки нельзя послушать. Их вид отличается от остальных, поймёте. 
Некоторые треки подменяют через mp3, загруженные пользователями. У таких может не быть альбома и они скрыты от публичного поиска.
